Resources/doc/commands.rst

```diff
@@ -0,0 +1,76 @@
+Commands
+========
+
+Remove cache
+------------
+
+All cache for a given paths will be removed:
+
+.. code-block:: bash
+
+    $ php app/console liip:imagine:cache:remove path1 path
+
+If you use ``--filters`` parameter, all cache for a given filters will be lost:
+
+.. code-block:: bash
+
+    $ php app/console liip:imagine:cache:remove --filters=thumb1 --filters=thumb2
+
+You can combine these parameters:
+
+.. code-block:: bash
+
+    $ php app/console liip:imagine:cache:remove path1 path2 --filters=thumb1 --filters=thumb2
+
+
+Cache for all paths and filters will be lost when executing this command
+without parameters:
+
+.. code-block:: bash
+    $ php app/console liip:imagine:cache:remove
+
+
+Resolve cache
+-------------
+
+.. code-block:: bash
+
+    $ php app/console liip:imagine:cache:resolve path1 path2 --filters=thumb1
+
+
+Cache for the two paths will be resolved using the passed filter.
+As a result you will get:
+
+.. code-block:: bash
+
+    http://localhost/media/cache/thumb1/path1
+    http://localhost/media/cache/thumb1/path2
+
+
+You can pass few filters:
+
+.. code-block:: bash
+
+    $ php app/console liip:imagine:cache:resolve path1 --filters=thumb1 --filters=thumb2
+
+
+As a result you will get:
+
+.. code-block:: bash
```

Comment:
For this kind of content, in Symfony Docs we use `.. code-block:: text` instead of `.. code-block:: bash`